src/current/v24.2/ui-overload-dashboard.md

```diff
@@ -15,93 +21,113 @@ To view this dashboard, [access the DB Console]({% link {{ page.version.version
+This graph shows the 99th percentile latency of requests waiting in the Admission Control elastic CPU queue, as tracked by the `admission.wait_durations.elastic-cpu-p99` metric.
 
-- In the node view, the graph shows the 75th percentile of latency for regular requests and elastic requests spent waiting for flow tokens on the selected node.
-- In the cluster view, the graph shows the 75th percentile of latency for regular requests and elastic requests spent waiting for flow tokens across all nodes in the cluster.
+- In the node view, the graph shows the delay duration of KV write on the selected node.
+- In the cluster view, the graph shows the delay duration of KV write across all nodes in the cluster.
 
-## Requests Waiting For Flow Tokens
+## Admission Queueing Delay p99 – Replication Admission Control
 
-This graph shows the number of regular requests and elastic requests waiting for flow tokens, as tracked respectively by the `cr.node.kvadmission.flow_controller.regular_requests_waiting` and the `cr.node.kvadmission.flow_controller.elastic_requests_waiting` metrics. There are separate lines for regular requests waiting and elastic requests waiting.
+This graph shows the 99th percentile latency of requests waiting in the Replication Admission Control queue, as tracked by the `kvadmission.flow_controller.regular_wait_duration-p99` and the `kvadmission.flow_controller.elastic_wait_duration-p99` metrics. There are separate lines for regular flow token wait time and elastic (background) flow token wait time. These metrics are indicative of store overload on replicas.
 
-- In the node view, the graph shows the number of regular requests and elastic requests waiting for flow tokens on the selected node.
-- In the cluster view, the graph shows the number of regular requests and elastic requests waiting for flow tokens across all nodes in the cluster.
+- In the node view, the graph shows the wait duration of regular flow token wait time and elastic flow token wait time on the selected node.
+- In the cluster view, the graph shows the wait duration of regular flow token wait time and elastic flow token wait time across all nodes in the cluster.
 
 ## Blocked Replication Streams
 
-This graph shows the number of replication streams with no flow tokens available for regular requests and elastic requests, as tracked respectively by the `cr.node.kvadmission.flow_controller.regular_blocked_stream_count` and the `cr.node.kvadmission.flow_controller.elastic_blocked_stream_count` metrics. There are separate lines for blocked regular streams and blocked elastic streams.
+This graph shows the blocked replication streams per node in Replication Admission Control, separated by admission priority {regular, elastic}, as tracked by the `kvadmission.flow_controller.regular_blocked_stream_count` and the `kvadmission.flow_controller.elastic_blocked_stream_count` metrics. There are separate lines for blocked regular streams and blocked elastic (background) streams.
```

Comment:
same as above re: adding link to 'operations in AC' subsection of AC overview page

do we have links to docs for these specific metrics? or is this page the doc for these metrics?  if we do have a page with docs for these metrics, it would be good if the metric names all became links to that metric in some table or list with specific docs for that metric, if such docs exist

  sorry, I realize I should have been asking this all throughout this page wherever a metric is mentioned